tests/testDefect.py

```diff
@@ -81,7 +80,9 @@ def testDefectBase(self):
         im = ccdImage.getImage()
         for d in defectList:
             intrp = im.Factory(im, d.getBBox())
-            self.assertImagesEqual(intrp, self.setVal)
+            expect = np.empty_like(intrp.getArray())
+            expect[:] = self.setVal
```

Comment:
I don't think it matters: I'm refilling it with self.setVal. I guess I'd rather just start with an uninitialized array, to ensure it doesn't accidentally use the values it wants to compare against.